.gitignore

```diff
@@ -56,3 +56,5 @@ target/
 *~
 *.swp
 site/
+
+.idea
```

Comment:
I think the current `.gitignore` is [the default Python `.gitignore`](https://github.com/github/gitignore/blob/master/Python.gitignore). I agree that it's not specific to our project.
I don't have a strong opinion either way. @awailly What do you think?